server/app.js

```diff
@@ -11,6 +12,7 @@ app.use(express.urlencoded({ extended: false }));
 
 // request mapping
 app.use('/contents', contentRouter);
+app.use('/collectionList', collectionRouter);
```

Comment:
```suggestion
app.use('/collections', collectionRouter);
```
라고 하는건 어때요? 위의 content는 contents라고 했으니 거기에 맞추는 것도 좋을듯

Reply:
네 좋아요 client쪽에서도 수정해야겠네요